roles/provision-buildfarm-apb/templates/android.configmap.yaml.j2

```diff
@@ -14,8 +14,8 @@ data:
       <privileged>false</privileged>
       <volumes>
         <org.csanchez.jenkins.plugins.kubernetes.volumes.PersistentVolumeClaim>
-          <mountPath>/opt/android-sdk-linux</mountPath>
-          <claimName>android-sdk</claimName>
+          <mountPath>{{ lookup('env', 'ANDROID_HOME')|default('/opt/android-sdk-linux', True) }}</mountPath>
```

Comment:
Can you use the default variable instead of  hard-coding the default `ANDROID_HOME` value: https://github.com/aerogearcatalog/aerogear-digger-apb/blob/master/roles/provision-buildfarm-apb/defaults/main.yml#L2?